test/app/app.test.js

```diff
@@ -137,28 +137,8 @@ describe('simple app', function () {
 				});
 
 		});
-
-		//fixme - I'm not sure ow this test ever passed but it doesn't now
-		it.skip('should notify sentry of unrecognised services', function (done) {
-
```

Comment:
I delete this test do I need a test for new code?

Reply:
no you can delete it safely. Wasn't running anyway